packages/calcite-components/src/components/action/action.tsx

```diff
@@ -313,10 +313,10 @@ export class Action
       <Host>
         <InteractiveContainer disabled={disabled}>
           <button
-            aria-busy={toAriaBoolean(loading)}
+            aria-busy={loading ? toAriaBoolean(loading) : null}
             aria-controls={indicator ? indicatorId : null}
             aria-label={ariaLabel}
-            aria-pressed={toAriaBoolean(active)}
+            aria-pressed={active ? toAriaBoolean(active) : null}
```

Comment:
@geospatialem I wonder if we need a new props like `pressable` and `loadable` in order to properly do this?

In theory, any action can be active or loading at any time so maybe this PR isn't valid? I'd love your thoughts here.